### PR TITLE
App 959 update readme salesforce

### PR DESCRIPTION
**Pull request by crepache**
Update README with MessageML v1 and v2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The data already comes in a Symphony MessageML format (as it is generated by our
 
 ### Opportunity Notification
```
Comment (robson-sanchez) [inline README.md:19]: You might identify that this event has two different formats.

Something like that: "The opportunity notification may have two different formats: XML or JSON"

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The data already comes in a Symphony MessageML format (as it is generated by our
...
 
-* Payload sent from Salesforce
+* Payload with content-type XML sent from Salesforce
```
Comment (robson-sanchez) [inline README.md]: * XML Payload

File: README.md [hunk 3]
```diff
@@ -65,10 +65,211 @@ The data already comes in a Symphony MessageML format (as it is generated by our
...
 </messageML>
 ```
-* Notification on Symphony platform
+
+##### Message rendered on Symphony
 
 ![Opportunity](src/docs/sample/sample_opportunity_rendered.png)
 
+
+* Payload with content-type JSON sent from Salesforce
```
Comment (robson-sanchez) [inline README.md]: * JSON Payload

**Review by robson-sanchez — CHANGES_REQUESTED** [review]
Small changes

File: README.md (hunk 2 above)
Comment (crepache) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (crepache) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (crepache) [reply README.md:19]: Done.

**Review by robson-sanchez — APPROVED** [review]

**Pull request merged**